### PR TITLE
MDA-style docs

### LINKED ISSUE #69: use MDA style for docs
It would look nice if the mdacli docs followed the style of the other docs to show that it's part of the MDA ecosystem and directly supported by the MDAnalysis org.

And a logo...

### PR DESCRIPTION
**Pull request by orbeckst**
- fix #69 
- update style
- create sitemap (assuming the docs will eventually be at https://www.mdanalysis.org/mdacli)
- add the default footer with the link to the privacy (**required for all MDA docs within the MDAnalysis organization**)
- add preliminary logo

### CODE REVIEW

**Comment by orbeckst** [comment]
I used the standard MDA styles to give you a starting point . I also created a simple logo as a placeholder.

**Comment by orbeckst** [comment]
Needs
- sphinx_rtd_theme
- sphinx_sitemap

**Review by joaomcteixeira — COMMENTED** [review]
Great addition. thanks! :wink:

I made some changes to pass CI, but not completed yet. Need to clean some details in the documentation.

`tox -e docs` tells what is missing.

**Comment by orbeckst** [comment]
@joaomcteixeira I am not sure what I could do to fix the remaining issuse. Can I leave this to you?

File: docs/rst/_templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "!footer.html" %}
+
+{% block extrafooter %}
+{{ super() }}
+<div class="footer"><p>Please see
+    our <a href="https://www.mdanalysis.org/pages/privacy/">Privacy Policy</a>
+    to learn how <a href="https://www.mdanalysis.org">MDAnalysis</a> collects data.</p>
+    <script data-goatcounter="https://mdanalysis.goatcounter.com/count"
+        async src="//gc.zgo.at/count.js"></script>
```
Comment (orbeckst) [inline docs/rst/_templates/footer.html:9]: Note that this part is necessary for analytics.

The number of page accesses is actually an important number for us to demonstrate impact so we do want to keep track of which pages are seen. This is fully anonymous, though.

File: docs/rst/conf.py [hunk 2]
```diff
@@ -33,8 +35,13 @@
     'sphinx.ext.todo',
     'sphinx.ext.viewcode',
     'sphinx.ext.autosectionlabel',
+    'sphinx_rtd_theme',
+    'sphinx_sitemap',
     ]
 
+# for sitemap with https://github.com/jdillard/sphinx-sitemap
+site_url = "https://www.mdanalysis.org/mdacli/"
```
Comment (orbeckst) [inline docs/rst/conf.py:43]: in preparation for #71

**Comment by PicoCentauri** [comment]
Thanks @orbeckst!

I fixed the CI (adding a required CHANGELOG entry) and updated the AUTHORS. LGTM. @joaomcteixeira  if you want to update the docs do it and merge when you think.

Just to know. For adding it to https://www.mdanalysis.org/mdacli we should update our github action for pushing it right?

**Comment by orbeckst** [comment]
> Just to know. For adding it to https://www.mdanalysis.org/mdacli we should update our github action for pushing it right?

Yes. You can definitely keep your RTD as well but when we refer to the docs we refer to the ones hosted under mdanalysis.org.

Once that's up, we add your mdacli/sitemap.xml to https://www.mdanalysis.org/sitemapindex.xml so that (1) search engines can find everything and (2) the search on the website itself will eventually find the docs (although  there are currently some issues with the search https://github.com/MDAnalysis/MDAnalysis.github.io/issues/202).

**Comment by PicoCentauri** [comment]
Okay! Sounds like a good plan. I suggest we create another PR for displaying the docs at mdanalysis.org including a sitemap and merge this one asap.

**Comment by orbeckst** [comment]
I switched the logo to the same one that you used in PR https://github.com/MDAnalysis/MDAnalysis.github.io/pull/207 ; from my end I think everything is done here. Now up to @PicoCentauri and @joaomcteixeira .

**Review by PicoCentauri — APPROVED** [review]

**Pull request merged**